x-pack/plugins/apm/public/components/alerting/TransactionDurationAlertTrigger/index.tsx

```diff
@@ -32,20 +29,38 @@ interface Params {
   environment: string;
 }
 
+const TRANSACTION_ALERT_AGGREGATION_TYPES = {
+  avg: i18n.translate(
+    'xpack.apm.transactionDurationAlert.aggregationType.avg',
+    {
+      defaultMessage: 'Average',
+    }
+  ),
+  '95th': i18n.translate(
+    'xpack.apm.transactionDurationAlert.aggregationType.95th',
+    {
+      defaultMessage: '95th percentile',
+    }
+  ),
+  '99th': i18n.translate(
+    'xpack.apm.transactionDurationAlert.aggregationType.99th',
+    {
+      defaultMessage: '99th percentile',
+    }
+  ),
+};
+
 interface Props {
-  alertParams: Params;
+  alertParams: AlertParams;
   setAlertParams: (key: string, value: any) => void;
   setAlertProperty: (key: string, value: any) => void;
 }
 
 export function TransactionDurationAlertTrigger(props: Props) {
   const { setAlertParams, alertParams, setAlertProperty } = props;
-  const { serviceName } = alertParams;
   const { urlParams } = useUrlParams();
-
   const transactionTypes = useServiceTransactionTypes(urlParams);
-
-  const { start, end } = urlParams;
+  const { serviceName, start, end, transactionType } = urlParams;
```

Comment:
In other triggers we use

`const { serviceName } = useParams<{ serviceName?: string }>();`

Should it be consistent?

Reply:
Fixed.